### DISCUSSION TITLE
SMB_tests fail - host and path seem to be merged into host.

### DISCUSSION THREAD

**Discussion by torstefan** [wifinigel/wiperf#58] [Bugs - I think I may have found a bug] [answered]
Hey

I get the following error when trying to use the SMB test in version 2.2.0 of wiperf

wiperf_agent.log.10:2021-03-30 16:01:47,708 - Probe_Log - ERROR - Hit an error with SMB mount pppwlanpi : mount error: could not resolve address for pppwlanpi: Unknown error

Its seems to be merging the smb_path1 and smb_hosts1 into smp_host1. 

Anyone else have this bug?

root@snorty-svg:/var/log# /usr/share/wiperf/setup.sh check_ver

Installed version: 2.2.0
Available version: 2.2.0

[SMB_test]
; yes = enabled, no = disabled
enabled: no
;
; Number of SMB targets we'd like to test
smb_targets_count: 1
;
; username and password to be used for all tests if all
; tests use same credentials
smb_global_username:
smb_global_password:
;
; first host we'd like to test
smb_host1: ppp
smb_username1: svc_hvi_wlanpi
smb_password1: xxx
smb_path1: wlanpi
smb_filename1: 100m.img

root@snorty-svg:/var/log# ping ppp
PING vir-app5660.xxx.net (xxx.xxx.8.50) 56(84) bytes of data.
64 bytes from VIR-APP5660.xxx.net (xxx.xxx.8.50): icmp_seq=1 ttl=122 time=7.02 ms

**Comment by wifinigel** [comment, marked as answer]
The mount path is missing a path root. Try smb_path1: /wlanpi

**Reply by torstefan** [reply]
Jupp, this worked. Thx.
